langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -15,11 +15,15 @@
 // under the License.
 
 import ballerina/jballerina.java;
+import ballerina/lang.regexp;
 
 # Built-in subtype of string containing strings of length 1.
 @builtinSubtype
 type Char string;
 
+//todo @chiran
+//type RegExp regexp:RegExp;
```

Comment:
Relates to https://github.com/ballerina-platform/ballerina-lang/issues/37892

Reply:
This is fixed by https://github.com/ballerina-platform/ballerina-lang/pull/37956/